### DISCUSSION TITLE
Export route name

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#348] [Q&A] [answered]
By the way is it still not solveable that the export name is always date and time and not the name of the before openened file?

_Originally posted by @DolomitesTom in https://github.com/devemux86/cruiser/discussions/107#discussioncomment-7305547._

**Comment by devemux86** [comment]
We had this in old versions?

I will have to check again the code. :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
Yes we had this in former versions, already in K1pro an K2... I think also in Cruiser, don't know when this changed... thank you.

**Comment by devemux86** [comment]
@DolomitesTom

Note that on Android in order to overwrite a file, _we must first click on the file_.
It does not happen automatically with same name, this is how Android works.

And when we click a file, its name is filled in automatically.

**Reply by DolomitesTom** [reply]
Yes but if you want to create a new file for example a variant name, it should already proposes the actual name. Less writing and if you delete the date and time over the 2  then it moves the folder to the parent folder.

**Reply by devemux86** [reply]
> if you delete the date and time over the 2 then it moves the folder to the parent folder

This seems to happen when we press delete while the name field is empty.
Shouldn't happen, probably an Android error for quite some time.

**Comment by devemux86** [comment]
The code was there but disabled! I enabled it again and now also works with the [favorite routes](https://github.com/devemux86/cruiser/discussions/35#discussioncomment-7036955).

You can test it with the latest:
- https://github.com/devemux86/cruiser/discussions/329

**Reply by DolomitesTom** [reply]
Unfortunately doesn't work, still date time as file name, even if a file is loaded before.

**Reply by devemux86** [reply]
@DolomitesTom do you use the b11?

Please describe your exact steps so I can reproduce what you are doing.

**Reply by DolomitesTom** [reply]
Yes B11. Just opened a locally stored kurviger Routing file, changed a WP and exported again. Even without changing something the proposed Export file name is date time.

**Reply by WolfgangZ42** [reply]
> Just opened a locally stored kurviger Routing file, changed a WP and exported again.

Strange - works fine in my environment ...

**Reply by 0709wiwiwi** [reply]
> The code was there but disabled!

_Presumably was disabled since Android provides a copy with (1) AFTER the file extension in (identical) file exports ?_
And by the way like @WolfgangZ42 I too can not confirm the reported issue by @DolomitesTom 
Tested multiple times as to be ok by using a gpx import and export.

**Reply by devemux86** [reply]
I tested it again and cannot find any problem:
- Create a random route in Kurviger website and export it: [Andorra la Vella.kurviger.txt](https://github.com/devemux86/cruiser/files/13044245/Andorra.la.Vella.kurviger.txt)
- In Cruiser use the menu "Routing | Import" or click the file in file manager with "Open with" dialog
- The route opens in Cruiser and in export the "Andorra la Vella" name is filled in all export screens

**Comment by DolomitesTom** [comment]
I see what the problem was. i imported as overlay (last setting), so it was not really a new loaded route. Sorry for confusion. When I open it as a route then it works. Thanks a lot. Only the other problem with the filter change still exists. Bad this is not a big issue if the name is proposed.

**Reply by devemux86** [reply]
@DolomitesTom 

Do you mean the folder change with backspace (as you described [above](https://github.com/devemux86/cruiser/discussions/348#discussioncomment-7306272))?
This is a strange Android thing...

**Reply by DolomitesTom** [reply]
Yes if tap and hold the backspace key for ex.

**Comment by devemux86** [comment, marked as answer]
Route name is again available in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
